e2-poster/styles/styles.scss

```
@@ -8,117 +8,560 @@
+        transform: rotate(340deg);
+        width: 0.4vw;
+        z-index: 10;
+    }
+
+    &--wrectb {
+        background-color: $white;
+        height: 2.7vw;
+        left: 41.5vw;
+        top: 92vh;
+        transform: rotate(330deg);
+        width: 0.5vw;
+        z-index: 10;
+    }
+
+    &--rtgl {
+        animation: floaty 60s infinite alternate;
+        _border-color: $black $black $red $black;
+        _filter: progid:DXImageTransform.Microsoft.Chroma(color="#$black");
+        border-color: transparent transparent $red transparent;
+        border-style: solid;
+        border-width: 0 12vh 2vh 2vh;
+        filter: drop-shadow(0.25vw 0.25vw $white);
+        left: 40vw;
+        line-height: 0;
+        top: 70vh;
+        transform: rotate(350deg);
+        z-index: 6;
+    }
+
+    &--btgl {
+        animation: floaty 60s infinite alternate;
+        _border-color: $black $black $black $black;
+        _filter: progid:DXImageTransform.Microsoft.Chroma(color="#$black");
+        border-color: transparent transparent $black transparent;
+        border-style: solid;
+        border-width: 0 15vh 1vh 1vh;
+        left: 39vw;
+        line-height: 0;
+        top: 70.5vh;
+        z-index: 6;
+    }
+
+    &--rrecta {
+        animation: floaty 60s infinite alternate;
+        background-color: $red;
+        filter: drop-shadow(0.25vw 0.25vw $white);
+        height: 1vh;
+        left: 20vw;
+        top: 78vh;
+        width: 10vw;
+        z-index: 4;
+    }
+
+    &--wbrecta {
+        animation: floaty 60s infinite alternate;
+        background-color: $white;
+        border: solid black 0.1vh;
+        height: 1vh;
+        left: 20vw;
+        top: 79vh;
+        width: 10vw;
+        z-index: 3;
+    }
+
+    &--rrectb {
+        animation: floaty 60s infinite alternate;
+        background-color: $red;
+        height: 10vh;
+        left: 28vw;
+        top: 82vh;
+        width: 2vw;
+        z-index: 3;
+    }
+
+    &--rrectc {
+        animation: floaty 60s infinite alternate;
+        background-color: $red;
+        height: 1vh;
+        left: 31vw;
+        top: 83.5vh;
+        width: 2vw;
+        z-index: 3;
+    }
+
+    &--rrectd {
+        animation: floaty 60s infinite alternate;
+        background-color: $red;
+        height: 0.8vh;
+        left: 31vw;
+        top: 86vh;
+        width: 2vw;
+        z-index: 3;
+    }
+
+    &--rrecte {
+        animation: floaty 60s infinite alternate;
+        background-color: $red;
+        height: 1vh;
+        left: 31vw;
+        top: 89vh;
+        width: 2vw;
+        z-index: 3;
+    }
 }
 
-.txt__krasnym {
-    font-size: 2em;
-    color: $white;
-    -webkit-text-fill-color: $white;
-    /* Will override color (regardless of order) */
-    -webkit-text-stroke-width: 0.033em;
-    -webkit-text-stroke-color: $black;
-    transform: rotate(28deg);
-    top: 55vh;
-    left: 7vw;
+.clutter__btm--v {
+
+    &a {
+        background: $white;
```

Written this way, this element requires knowledge of the background it exists on top of. While this works, it can create maintenance problems. What about:
```suggestion
        background: transparent;
```